javaslang/generator/Generator.scala

```diff
@@ -1357,6 +1402,26 @@ def generateTestClasses(): Unit = {
 
       im.getStatic("javaslang.API.*")
 
+      def genAliasesTests(im: ImportManager, packageName: String, className: String): String = {
+        xs"""
+          ${(0 to N).gen(i => {
+            val params = (1 to i).gen(j => s"v$j")(", ")
+            xs"""
+              @$test
+              public void shouldFunction${i}ReturnNotNull() {
+                  assertThat(function$i(($params) -> null)).isNotNull();
```

Comment:
Almost all current tests created only for test what alias return something, and only few tests check that alias return exactly that he need to return (Future, Try).